checkov/common/bridgecrew/integration_features/features/policy_metadata_integration.py

```diff
@@ -113,15 +120,25 @@ def _handle_public_metadata(self, check_metadata):
 
     def _handle_customer_run_config(self, run_config):
         self.check_metadata = run_config['policyMetadata']
-        self.bc_to_ckv_id_mapping = {pol['id']: ckv_id for (ckv_id, pol) in self.check_metadata.items()}
-
+        for (ckv_id, pol) in self.check_metadata.items():
+            self.bc_to_ckv_id_mapping[pol['id']] = ckv_id
+            if self.bc_integration.is_prisma_integration() and pol.get('pcPolicyId'):
+                self.pc_to_ckv_id_mapping[pol['pcPolicyId']] = ckv_id
         # Custom policies are returned in run_config['customPolicies'] rather than run_config['policyMetadata'].
         if 'customPolicies' in run_config:
             for custom_policy in run_config['customPolicies']:
                 if 'guideline' in custom_policy:
                     self.check_metadata[custom_policy['id']] = {
                         'guideline': custom_policy['guideline']
                     }
+                if custom_policy.get('pcPolicyId'):
+                    self.pc_to_ckv_id_mapping[custom_policy['pcPolicyId']] = custom_policy['id']
+
+    def _handle_customer_prisma_policy_metadata(self, prisma_policy_metadata):
+        if isinstance(prisma_policy_metadata, list):
```

Comment:
The API only returns a list and if it doesn't for some reason, it would indicate there was an error and the result will be filtering is skipped. So I think this should be alright!